docs/guides/fine_tuning.md

```diff
@@ -152,7 +152,15 @@ See the [Model Zoo](../../model_zoo) to see which models have fine-tuning suppor
 
 See [Integrations](../integrations.md) to see how to track fine-tuning metrics.
 
-Once the fine-tune is launched, you can also [get the status of your fine-tune](../../api/python_client/#llmengine.fine_tuning.FineTune.get). You can also [list events that your fine-tune produces](../../api/python_client/#llmengine.fine_tuning.FineTune.get_events).
+## Monitoring the fine-tune
+
+Once the fine-tune is launched, you can also [get the status of your fine-tune](../../api/python_client/#llmengine.fine_tuning.FineTune.get). 
```

Comment:
This is great, but can we sprinkle code snippets for examples on how to do various things in here?

Would also be good to have snippets of error examples. Basically we're trying to make this more tangible with examples.